crates/forge/tests/e2e/diagnostics_and_plugins.rs

```diff
@@ -37,7 +42,7 @@ fn print_error_if_attributes_incorrect() {
              --> test_fuzzer.cairo:8:9
             #[fuzzer()]
                     ^^
-
+            
```

Comment:
👀

Reply:
Was just a newline here before and I used tab on multiple lines, it doesn't change anything